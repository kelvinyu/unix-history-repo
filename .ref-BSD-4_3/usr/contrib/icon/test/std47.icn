record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("2 === +2 ----> ",image(2 === +2) | "none")
   write("2.0 === +2.0 ----> ",image(2.0 === +2.0) | "none")
   write("3 === *\"abc\" ----> ",image(3 === *"abc") | "none")
   write("'abc' === ('abc' ++ '') ----> ",image('abc' === ('abc' ++ '')) | "none")
   write("'abc' = ('abc' ++ '') ----> ",image('abc' = ('abc' ++ '')) | "none")
end
