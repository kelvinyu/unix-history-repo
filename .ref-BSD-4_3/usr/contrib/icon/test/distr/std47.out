2 === +2 ----> 2
2.0 === +2.0 ----> 2.0
3 === *"abc" ----> 3
'abc' === ('abc' ++ '') ----> 'abc'
Run-time error 102 at line 13 in std47.icn
numeric expected
offending value: "abc"
